src/draft4.json

```diff
@@ -1,224 +1,149 @@
 {
+    "id": "http://json-schema.org/draft-04/schema#",
```

Comment:
Can we document where this file and the other json files in `src/` were pulled from? It's not obvious to me how we update that.

Also ... why are there updates to an older draft? I would have expected those to be stable and only the latest draft getting updates (or a new draft version to be released)

Reply:
Documenting -- absolutely.  Added some text to the README.

I don't know why there were updates to an earlier draft.  I only occasionally lurk in the JSON schema discussions, and I think they largely still treat this as if they are in semver 0.x, even though this has been going on for many years.

Apparently this practice has [recently changed](http://json-schema.org/specification-links.html), so this is hopefully the last time we'll need to do something like this:

> The practice of fixing meta-schema bugs in place to preserve the sequential numbering has also been controversial.

> Starting with what had been called “Draft 8” while it was being written, meta-schemas are identified by the year and month of publication. This allows for bug fixes to be published with new URIs (as long as it is not more than once a month).

Also note -- some of these changes are to the meta schema, but the changes to our code here were required because the test coverage has improved -- mainly around testing Javascript numeric comparison semantics and `$id`-based cross-referencing.  Fortunately for us, Glean uses neither of these features, but better to have this compliant in advance of any of those features being used.